### PR TITLE
docs: add rest api

### PR DESCRIPTION
**Pull request by Samk13**

### CODE REVIEW

**Comment by Samk13** [comment]
@matves29  Could you please review the language?

File: docs/en/deposit/rest_api.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+```
+
+Example Request:
+A basic example request to fetch posts from the KTH Datarepository instance could be:
+
+```bash
+GET /api/records?prettyprint=1 HTTP/1.1
+```
+
+This is the basic information about the KTH Datarepository API. However, for a complete API reference, it is recommended to refer to the official documentation, as it may contain more details and the latest updates:
```
Comment (matves29) [inline docs/en/deposit/rest_api.md:40]:
```suggestion
This is the basic information about the KTH Datarepository API. However, for a complete API reference, it is recommended to refer to the official documentation, as it contains more details and the latest updates:
```

File: docs/en/deposit/rest_api.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Authentication
+
+The REST API calls for KTH Datarepository require authentication. The currently supported authentication method is through the use of Bearer tokens, which can be generated in the "Applications" section of your user account's settings in your KTH Datarepository instance.
```
Comment (matves29) [inline docs/en/deposit/rest_api.md:3]:
```suggestion
The REST API calls for KTH Datarepository require authentication. The currently supported authentication method is through the use of Bearer tokens, which can be generated in the "Applications" section of your user account's settings in the KTH Datarepository.
```

File: docs/sv/deposit/rest_api.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+
+# Autentisering
+
+REST API-anrop för KTH Datarepository kräver autentisering. Den för närvarande stödda autentiseringsmetoden är genom användning av Bearer-token, som kan genereras i "Applikationer" -sektionen av dina användarkontoinställningar i din KTH Datarepository-instans.
```
Comment (matves29) [inline docs/sv/deposit/rest_api.md:4]:
```suggestion
REST API-anrop för KTH Datarepository kräver autentisering. Den för närvarande stödda autentiseringsmetoden är genom användning av Bearer-token, som kan genereras i "Applikationer" -sektionen av dina användarkontoinställningar i KTH Datarepository.
```

File: docs/sv/deposit/rest_api.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+curl -k -H "Authorization: Bearer API-TOKEN" https://datarepository.kth.se/api/records
+```
+
+- access_token HTTP-frågesträngparameter:
+
+```bash
+curl -k https://datarepository.kth.se/api/records?access_token=API-TOKEN
+```
+
+## Räckvidder
```
Comment (matves29) [inline docs/sv/deposit/rest_api.md:20]:
```suggestion
## Omfattning
```

File: docs/sv/deposit/rest_api.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+När du skapar ditt API-token kan du specificera räckvidder som kontrollerar vilken typ av resurser och åtgärder du kan komma åt med ditt token. De tillgängliga räckvidderna för genererade tokens kan ändras när åtkomstkontrollmekanismerna till poster är slutgiltiga.
```
Comment (matves29) [inline docs/sv/deposit/rest_api.md:22]:
```suggestion
När du skapar ditt API-token kan du specificera omfattning som kontrollerar vilken typ av resurser och åtgärder du kan komma åt med ditt token. De tillgängliga omfattningarna för genererade tokens kan ändras när åtkomstkontrollmekanismerna till poster är slutgiltiga.
```

Comment (matves29) [reply docs/sv/deposit/rest_api.md:22]: this is a bit convuluted in both english and swedish but ok.

File: docs/sv/deposit/rest_api.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+### Allmän information
+
+**Tidsstämplar:** KTH Datarepository tillhandahåller tidsstämplar i UTC formaterade enligt ISO 8601.
+
+**Pretty print JSON:** Om du utforskar API:et via en webbläsare kan du formatera JSON genom att lägga till prettyprint=1 i frågesträngen. Till exempel:
```
Comment (matves29) [inline docs/sv/deposit/rest_api.md:28]:
```suggestion
**Pretty print JSON:** Om du kopplar upp mot API:et via en webbläsare kan du formatera JSON genom att lägga till prettyprint=1 i frågesträngen. Till exempel:
```

File: docs/sv/deposit/rest_api.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+### Exempelförfrågan
+
+En grundläggande exempelförfrågan för att hämta poster från KTH Datarepository-instansen kan vara:
+
+```bash
+GET /api/records?prettyprint=1 HTTP/1.1
+```
+
+Detta är grundläggande information om KTH Datarepository API. Men för en komplett API-referens rekommenderas det att hänvisa till den officiella dokumentationen, eftersom den kan innehålla mer detaljer och de senaste uppdateringarna:
```
Comment (matves29) [inline docs/sv/deposit/rest_api.md:42]:
```suggestion
Detta är grundläggande information om KTH Datarepository API. Men för en komplett API-referens rekommenderas den officiella dokumentationen, eftersom den innehåller mer detaljer och de senaste uppdateringarna:
```

File: docs/sv/deposit/restrict_record_access.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
+
+# Skapa en begränsad post
+
+Denna tjänst erbjuder en funktionalitet för att skapa poster med begränsad åtkomst. Denna funktion tillåter datakuratorer och administratörer att begränsa synligheten för vissa poster, vilket gör dem tillgängliga endast för behöriga användare. I den här guiden beskriver vi processen för att skapa en begränsad post.
```
Comment (matves29) [inline docs/sv/deposit/restrict_record_access.md:4]:
```suggestion
Denna tjänst erbjuder en funktionalitet för att skapa poster med begränsad åtkomst. Denna funktion tillåter datagranskare och administratörer att begränsa synligheten för vissa poster, vilket gör dem tillgängliga endast för behöriga användare. I den här guiden beskriver vi processen för att skapa en begränsad post.
```

File: docs/sv/deposit/restrict_record_access.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+
+## Förutsättningar
+
+- Du måste vara inloggad för att kunna skicka in din post.
+
+## Steg för att skapa en begränsad post
+
+1. ### Logga in på ditt konto
+
+   Navigera till inloggningssidan och ange dina referenser.
```
Comment (matves29) [inline docs/sv/deposit/restrict_record_access.md:16]:
```suggestion
   Navigera till inloggningssidan och logga in.
```

File: docs/sv/deposit/restrict_record_access.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+
+3. ### Fyll i postuppgifter
+
+   Fyll i formuläret med nödvändig information om din post, såsom titel, författare och annan metadata.
+
+4. ### Ställ in åtkomstbegränsningar
+
+   Leta efter alternativen till höger på insättningssidan eller avsnittet märkt "Synlighet". Här kommer du att kunna:
+   - **Ställ in fullständig postsynlighet:** Välj "Begränsad" för att begränsa synligheten för postens metadata och filer tillsammans.
+   - **Ange endast filer:** Välj "Begränsad" för att begränsa synligheten för endast postfiler, medan metadata förblir offentliga.
```
Comment (matves29) [inline docs/sv/deposit/restrict_record_access.md:30]:
```suggestion
   - **Ange endast filer:** Välj "Begränsad" för att begränsa synligheten för endast postfiler, medan metadata förblir offentlig.
```

File: docs/sv/deposit/restrict_record_access.md [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+
+5. ### Skicka in för granskning och bekräfta
+
+   När du har fyllt i insättningsformuläret klickar du på knappen "Skicka för granskning". Din begränsade post kommer nu att skapas för granskning av communitykuratorerna, och du kommer att meddelas när du accepteras genom att kontrollera din inkorgsknapp under "/me/requests"-rutt.
```
Comment (matves29) [inline docs/sv/deposit/restrict_record_access.md:34]:
```suggestion
   När du har fyllt i formuläret klickar du på knappen "Skicka för granskning". Din begränsade post kommer nu att skickas för granskning av communityns granskare, och du kommer att meddelas när du accepteras genom att kontrollera din inkorg, under "/me/requests".
```

File: docs/sv/deposit/restrict_record_access.md [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+
+## Ställa in en publicerad post till begränsad i KTH Datarepository
+
+Om du vill ändra en redan publicerad post till en begränsad status, följ dessa steg:
```
Comment (matves29) [inline docs/sv/deposit/restrict_record_access.md:38]:
```suggestion
Om du vill ändra en redan publicerad post till begränsad, följ dessa steg:
```

File: docs/sv/deposit/share_record_access.md [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+
+   Från din instrumentpanel eller postlista, lokalisera och klicka på din post som du vill dela.
+
+2. ### Leta upp delningsalternativet
+
+   I postens detaljerade vy hittar du "Dela" under redigera och ny version knapparna till höger på skärmen.
+
+3. ### Dela länken
+
+   När du klickar på knappen "Dela" bör ett modal visas "Hämta en länk" som ger en direktlänk till din post. Tryck på Hämta länk, vänta en sekund, sedan kopieras länken som genereras automatiskt.
```
Comment (matves29) [inline docs/sv/deposit/share_record_access.md:22]:
```suggestion
   När du klickar på knappen "Dela" visas "Hämta en länk" som ger en direktlänk till din post. Tryck på Hämta länk, vänta en sekund, sedan kopieras länken som genereras automatiskt.
```

**Review by matves29 — COMMENTED** [review]
ok with me!

**Pull request merged**